pkg/providers/vault/README.md

```diff
@@ -7,26 +7,49 @@ Vault and use the Secret Store CSI driver interface to mount them into Kubernete
 
 ## Demo
 
-TODO
+![Secret Store CSI Driver Vault Provider Demo](./images/secret-store-csi-driver-vault-provider-demo.gif "Secret Store CSI Driver Vault Provider Demo")
 
 ## Prerequisites
 
 The guide assumes the following:
 
-* A Kubernetes cluster up and running.
-* A Vault cluster up and running. Instructions for spinning up a Vault cluster in Kubernetes can be
-found [here](#TODO).
+* A Kubernetes cluster version 1.13.x+ up and running.
```

Comment:
Should we point users to use v1.15.0-alpha2+ for inline volume? to be consistent with https://github.com/deislabs/secrets-store-csi-driver#mount-secret-data-to-resource-through-inline-volume